Source/diablo.cpp

```diff
@@ -140,8 +156,8 @@ const char *const spszMsgTbl[] = {
 	N_("Here's something for you."),
 	N_("Now you DIE!")
 };
-/** INI files variable names for quick message keys */
-const char *const spszMsgHotKeyTbl[] = { "F9", "F10", "F11", "F12" };
+/** INI files variable names for quick messages */
+const char *const spszMsgNameTbl[] = { "QuickMessage1", "QuickMessage2", "QuickMessage3", "QuickMessage4" };

```

Comment:
I'm a bit conflicted with this one as it break compact with the original ini, but the names are pretty stupid

Reply:
Yeah, but I could not find a better way to do it.

Reply:
Let go with your solution.